datasets/ro/onpcsb_sanctions/ro_onpcsb_sanctions.yml

```diff
@@ -0,0 +1,41 @@
+title: "Romania Government Decision No. 1.272/2005: List of Suspected Terrorists"
+url: https://www.onpcsb.ro/en/a/126/legislatie
+entry_point: crawler.py
+prefix: ro-onpcsb
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+coverage:
+  frequency: never
+  start: 2024-07-24
+summary: >
+  The dataset contains information about suspected individuals and organizations involved in terrorism, approved by the Government of Romania
```

Comment:
it's nice if the text in these files can be wrapped roughly around 90 chars wide. Just makes it easier to edit in smaller windows, e.g. split screen.